CanvasColorSpaceProposal.md

```diff
@@ -100,6 +100,7 @@ The pixelFormat attributes specifies the numeric types to be used for storing pi
 * When an unsupported format is requested, the format shall fall back to "8-8-8-8".
 * With formats that use integer numeric types for the color channels, the canvas pixel buffer shall store gamma-corrected color values, using the transfer curves prescribed by the specification of the current color space.
 * With formats that use floating-point numeric types for the color channels, the canvas pixel buffer shall store non-gamma-corrected (a.k.a linear) color values.
+* Float values outside of [0,1] range can be used to represent colors outside of the chosen color gamut. This allows float pixel formats to represent all possible colors and brightness levels. How values outside of [0,1] are displayed depends on the capabilites of the device and output display. Some implementations may simply clamp these values to [0,1]. If the device and display are capable, a pixel value of (2,2,2) should be twice as bright as (1,1,1).
```

Comment:
What about alpha? Should we clamp it?  If not sure, put it in the open issues section.